packages/preview-middleware-client/src/adp/quick-actions/fe-v2/lr-enable-semantic-date-range-filter-bar.ts

```diff
@@ -25,7 +25,8 @@ export class ToggleSemanticDateRangeFilterBar
 
     async initialize(): Promise<void> {
         const isUI5VersionNotSupported = await areManifestChangesSupported();
-        if (isUI5VersionNotSupported) {
+        const pagesStructureInManifest = this.context.manifest['sap.ui.generic.app'].pages;
+        if (isUI5VersionNotSupported || Array.isArray(pagesStructureInManifest)) {
```

Comment:
This check should be part of `areManifestChangesSupported` function, so we have one function that we can use to check if manifest changes can be used.

Reply:
done

Reply:
We shouldn't use `not` in function names. In this particular case we need the inverse value, but it is quite likely that we would also need to check if the manifest changes are supported and we would need to use double negation, which is just looks weird and is more confusing `!areManifestChangesNotSupported()`. `!areManifestChangesSupported()` is a lot easier to read and understand.